tests/test_fastapi.py

```diff
@@ -0,0 +1,37 @@
+from typing import List
+
+import fastapi
+import pydantic
+import pytest
+import starlette.testclient
+
+PATH = "/"
+ROUTE = {"path": PATH, "endpoint": lambda: {}}
+
+
+@pytest.fixture(name="client")
+def fixture_client():
+    app = fastapi.FastAPI()
+    return starlette.testclient.TestClient(app)
+
+
+@pytest.fixture(
+    name="valid_responses",
+    params=[int, List[int], pydantic.BaseModel, List[pydantic.BaseModel]],
+    ids=["int", "List[int]", "pydantic.BaseModel", "List[pydantic.BaseModel]"],
+)
+def fixture_valid_responses(request):
+    return {"500": {"model": request.param}}
+
+
+def test_additional_responses_documented_in_openapi_schema(client, valid_responses):
+    """
+    The only stated guarantee w/ additional responses is that they'll be included in the
+    auto-generated OpenAPI docs -- so test that guarantee.
```

Comment:
I _think_ this is true, based on [the docs](https://github.com/tiangolo/fastapi/blob/9c3c9b6e78768374868d690bc05918d58481e880/docs/advanced/additional-responses.md#additional-response-with-model):

> Those additional responses will be included in the OpenAPI schema, so they will also appear in the API docs.

Reply:
Yep, that's correct. I just updated it a bit to keep the style of the tests consistent with the tests in other places.